contracts/AINFTFactory.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/interfaces/IERC721.sol";
+
+import "./AINFT721.sol";
+
+contract AINFTFactory is Ownable {
+    event CloneAINFT721(address indexed originalNFT, address indexed clonedAINFT);
+    event CreateAINFT721(address indexed createdAINFT);
+
+    constructor() Ownable() {}
```

Comment:
should we limit the callable only for contract owner?

Reply:
It is a complex problem related to the existence of AINFTFactory contract itself. Let's discuss on #6 !